browser/ui/browser_commands.cc

```diff
@@ -683,6 +684,7 @@ void BringAllTabs(Browser* browser) {
 
   // Find all browsers with the same profile
   std::vector<Browser*> browsers;
+  base::flat_set<Browser*> browsers_to_close;
```

Comment:
I think it'd be okay to use flat_set this time, as we insert `browser*` conditionally 😄